### DISCUSSION TITLE
Please communicate in Chinese or English

### DISCUSSION THREAD

**Discussion by ZhouYixun** [SonicCloudOrg/sonic-server#284] [Announcements]
Sonic is a platform that integrates remote control debugging and automated testing of mobile devices, and strives to create a better use experience for global developers and test engineers.

If you want to participate, welcome to join us! 💪

If you want to support, you can give me a star. ⭐

**Comment by calvinxiong** [comment]
could you provide the function of borrowing and returning the phone by scanning a code?
At the same time, all mobile phone can be managed.

**Comment by ZhouYixun** [comment]
I'm very sorry that we don't have this plan in the short term.

**Comment by zyc123888** [comment]
部署使用，要求的服务器最低配置是多少？

**Comment by wan88888** [comment]
请问Sonic git flow 插件什么时候支持？

**Comment by ZhouYixun** [comment]
有的，你可以看看官网的jenkins插件，也支持pipeline

**Comment by jixianglhy** [comment]
设备识别成功，控件识别成功，但用例执行的时候报：
异常信息： org.cloud.sonic.driver.common.tool.SonicRespException: Cannot invoke "org.openqa.selenium.chrome.ChromeDriver.findElement(org.openqa.selenium.By)" because "this.chromeDriver" is null
这怎么解决呢
